src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -61,8 +61,8 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
     const {translate} = useLocalize();
     const StyleUtils = useStyleUtils();
     const currentUserPersonalDetails = useCurrentUserPersonalDetails();
-    const [expensifyCardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`);
-    const [allCardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}`);
+    const [allCardsList] = useOnyx(`${ONYXKEYS.CARD_LIST}`);
```

Comment:
@VickyStash why was this change necessary?
```
    const [allCardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}`);
    const [allCardsList] = useOnyx(`${ONYXKEYS.CARD_LIST}`);
```

Reply:
@koko57 On the members details page, we need to show not only expensify cards, but also custom feeds/direct feeds cards.
They are not stored in the `cards_` right away, we need to do API calls for every feed type to get the data, which can be a pretty big amount of calls.

So I've [asked](https://callstack-hq.slack.com/archives/C06ML6X0W9L/p1728482728126749?thread_ts=1727729604.987049&cid=C06ML6X0W9L) @robertjchen about the way I can get all user-related cards at once. And he [suggested](https://callstack-hq.slack.com/archives/C06ML6X0W9L/p1728544957562749?thread_ts=1727729604.987049&cid=C06ML6X0W9L) going with `cardList_` key.

So that's the reason of the update.